contracts/near/Cargo.toml

```diff
@@ -22,12 +22,12 @@ borsh = "0.9.3"
 rlp = "0.5.2"
```

Comment:
We also should update the project versions in all Cargo tomls

Reply:
6b5872ea69ebf30f9235eb3dc4305fee6eae275e